MetaMask/metamask-extension
# UX: Multichain: Send Flow: Move connected components to individual components

## **Description**
Having all of the connected components pulling data in the same component (`SendPage`) was a major, major performance drain that was causing the `SendPage` to re-render over and over again.  This PR separates out the children into their own components to limit re-renders.

## **Manual testing steps**

1.  Run `MULTICHAIN=1 yarn start`
2. Go to the Send page
3. See the new send page layout, as was there before.

## **Screenshots/Recordings**

<img width="399" alt="SCR-20231023-lvwu" src="https://github.com/MetaMask/metamask-extension/assets/46655/6e7d06cd-70e7-42a3-be0b-6eab1aff9341">


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
